### PR TITLE
Bugfix/dot env files on macos

### LINKED ISSUE #35: Can't select .env file on MAC OS
The VSIX environment file selector does not work (also if hidden files are visible) on MAC OS. A viable solution is to remove the filter in the method "chooseComposeEnvFile".

### PR DESCRIPTION
**Pull request by llakie**
Fixes #35 and #83

### CODE REVIEW

File: src/sdk/vscode/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 ## [Unreleased]
 
+- 0.9.6 [2021-06-07]
```
Comment (JochenKienzle) [inline src/sdk/vscode/CHANGELOG.md]: I think that has to be merged with the other PR of you

**Review by JochenKienzle — COMMENTED** [review]
There are some overlaps (e.g. this one comment) with the other PRs which have to be merged afterwards. So, no real findings beside the upcoming conflicts with the other PRs

File: src/sdk/vscode/CHANGELOG.md (hunk 1 above)
Comment (JochenKienzle) [reply src/sdk/vscode/CHANGELOG.md]: Now, the conflicts are there ;-)

Comment (llakie) [reply src/sdk/vscode/CHANGELOG.md]: Conflict resolved :) Was a cross correlation

**Review by JochenKienzle — APPROVED** [review]

**Pull request merged**